frontend/src/App.tsx

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import axios from "axios";
+import { useState } from "react";
+const serverURL = 'http://localhost:8000'
 
 function App() {
+  const [message, setMessage] = useState("");
```

Comment:
nice use of hooks! this is optional but for typescript if you want to be extra safe with typings you can rewrite this line to be

```
const [message, setMessage] = useState<string>("");
```

additionally, this isn't super important for our use case but there's definitely room for debate as to whether you want to use `""` as the default message or something more explicit like `null` or `undefined`. the former keeps the types more consistent but the latter is preferred stylistically in some places to avoid situations like `""` not being a valid input but legally is a string

Reply:
Gotcha, I changed the hook to `useState<boolean>(false)`.

Reply:
Clarification question: Do you want me to do `useState<boolean>(false)`? I find `useState<string>("")` to be simple and effective just for this temporary code.

Reply:
yeah you could've definitely used the ("") option. just wanted to point it out that it is something people give feedback on, depending on the style of the codebase